source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Tests/Domain/Dtos/ChargeCommands/Validation/InputValidation/ValidationRules/ChargePriceMaximumDigitsAndDecimalsRuleTests.cs

```diff
@@ -59,9 +60,41 @@ public void ValidationRuleIdentifier_ShouldBe_EqualTo(ChargeCommandBuilder build
             sut.ValidationRuleIdentifier.Should().Be(ValidationRuleIdentifier.ChargePriceMaximumDigitsAndDecimals);
         }
 
+        [Theory]
+        [InlineAutoDomainData(ValidationRuleIdentifier.ChargePriceMaximumDigitsAndDecimals)]
+        public void TriggeredBy_ShouldCauseCompleteErrorMessages_ToMarketParticipant(
+            ValidationRuleIdentifier validationRuleIdentifier,
+            ChargeCommandBuilder builder,
+            ILoggerFactory loggerFactory,
+            CimValidationErrorTextProvider cimValidationErrorTextProvider)
+        {
+            // Arrange
+            var invalidCommand = CreateInvalidCommand(builder);
+            var expectedPoint = invalidCommand.ChargeOperation.Points[0];
+            var triggeredBy = expectedPoint.Position.ToString();
+
+            // Act & arrange
+            var sutOne = new ChargePriceMaximumDigitsAndDecimalsRule(invalidCommand);
+            var sutTwo = new ChargeCimValidationErrorTextFactory(cimValidationErrorTextProvider, loggerFactory);
+
+            var errorMessage = sutTwo.Create(
```

Comment:
How about aligning actual vs expected naming?

```suggestion
            var actual = sutTwo.Create(
```